Backend/bicep/main.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "environmentName": {
```

Comment:
This is the standard way the azd samples fetch params. We can take or leave it but I've added it for now.